kafka_quota_enforcer/src/test/java/com/tesla/data/quota/enforcer/QuotaEnforcerTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.quota.enforcer;
+
+import static java.util.Collections.emptyList;
+import static org.mockito.Matchers.anyCollection;
+import static org.mockito.Matchers.anyCollectionOf;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class QuotaEnforcerTest {
+
+  private QuotaService quotaService;
+  private static final List<ConfiguredQuota> quotas = Arrays.asList(
+      new ConfiguredQuota("user1", "clientA", 5000, 4000, 100),
+      new ConfiguredQuota("<default>", null, 6000, 3000, null)
+  );
+
+  @Before
+  public void setup() {
+    quotaService = mock(QuotaService.class);
+  }
+
+  @Test
+  public void testCreate() {
+    QuotaEnforcer enforcer = new QuotaEnforcer(emptyList(), quotaService, true, false);
+    enforcer.create(quotas);
+    verify(quotaService, times(1)).create(quotas);
```

Comment:
nit: times(1) is the default with verify(), you can just drop the times parameter